torch/csrc/inductor/aoti_torch/shim_common.cpp

```diff
@@ -182,6 +185,186 @@ AOTITorchError aoti_torch_create_tensor_from_blob(
   });
 }
 
+static AOTITorchError aoti_torch__scaled_dot_product_flash_attention_internal(
```

Comment:
Nit: move this to the anonymous namespace.

Reply:
I think ```static``` makes internal linkage. But Yes, I will move it to the anon namespace for consistency.